lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -952,6 +953,17 @@ def _create_deployment_parameters(
             parameters=parameters,
         )
 
+        # dump arm_template and arm_parameters to file
+        dump_path = constants.RUN_LOCAL_PATH / "arm_dump"
```

Comment:
The path should be from environment, because there are many environment deployed in a test run. If so, those files can be put under the root path of environment path. So the dump_path is not necessary, just use `environment.log_path` in following file paths.